indico/modules/events/templates/display/administrative/event.html

```diff
@@ -0,0 +1,100 @@
+{% from 'events/display/indico/_common.html' import render_datetime %}
+{% from 'events/display/common/_manage_button.html' import render_manage_button %}
+{% from 'events/timetable/display/administrative/_common.html' import render_location %}
+
+{% macro render_event_time(event, timezone) %}
+    {% if event.start_dt.astimezone(timezone).date() == event.end_dt.astimezone(timezone).date() %}
+        {{ event.start_dt | format_date(format='full', timezone=timezone|string) }}
+        - <em class="time">{{ event.start_dt | format_time(format='medium', timezone=timezone) }}</em>
+    {% else %}
+        {% trans start_date=event.start_dt|format_date(format='full', timezone=timezone|string),
+                 start_time=event.start_dt|format_time(format='medium', timezone=timezone),
+                 end_date=event.end_dt|format_date(format='full', timezone=timezone|string),
+                 end_time=event.end_dt|format_time(format='medium', timezone=timezone) %}
+            from {{start_date}} ({{start_time}}) to {{end_date}} ({{end_time}})
+        {% endtrans %}
+    {% endif %}
+{% endmacro %}
+
+<table class="eventWrapper">
+    <tr>
+        <td>
+            <div class="manage-button-container">
+                {{ render_manage_button(event, 'EVENT', toggle_notes=false, show_note_operations=true) }}
+            </div>
+            <div class="reportNumberHeader">
+                {% for ref in event.references %}
+                    {{ ref.value }}<br>
+                {% endfor %}
+            </div>
+            {% block header %}
+                <br>&nbsp;<br>
+                <div class="eventHeader">
+                    {{ theme_settings.title_fr or
+                       'ORGANISATION EUROP&Eacute;ENNE POUR LA RECHERCHE NUCL&Eacute;AIRE' | safe }}
+                    <br>
+                    <span class="CERNTitle"> CERN </span>
+                    {{ theme_settings.title_en or
+                       'EUROPEAN ORGANIZATION FOR NUCLEAR RESEARCH' | safe }}
```

Comment:
Just tested it, you do need parentheses:

```
>>> render_template_string('{{ "foo" or "bar" | tojson }}')
u'foo'
>>> render_template_string('{{ ("foo" or "bar") | tojson }}')
u'"foo"'
```

However, would you actually use html entities in the settings instead of just using the unicode/UTF8 character? (FWIW, I would also not use the html entity but the actual character in the french title)